project/plugins.sbt

```diff
@@ -12,7 +12,7 @@ resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.14")
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.2.9")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.2.10-SNAPSHOT")
```

Comment:
can we ask for a release instead? sbt is having issues resolving plugins (we're getting rate limited AFAIR still...), so a stable version here would be better